wibotic_connector_can/include/wibotic_connector_can/wibotic_can_driver.hpp

```diff
@@ -0,0 +1,173 @@
+// Copyright 2024 Husarion sp. z o.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef WIBOTIC_CONNECTOR_CAN_WIBOTIC_CAN_DRIVER_HPP_
+#define WIBOTIC_CONNECTOR_CAN_WIBOTIC_CAN_DRIVER_HPP_
+
+#include <memory>
+#include <queue>
+
+#include <uavcan/helpers/ostream.hpp>
+#include <uavcan_linux/uavcan_linux.hpp>
+
+#include "wibotic_connector_can/uavcan_types/wibotic/WiBoticInfo.hpp"
+
+namespace wibotic_connector_can
+{
+
+/**
+ * @brief Abstract interface for the Wibotic CAN driver.
+ */
+class WiboticCanDriverInterface
+{
+public:
+  /**
+   * @brief Virtual destructor for the WiboticCanDriverInterface class.
+   */
+  virtual ~WiboticCanDriverInterface() = default;
+
+  /** *
+   * @param can_iface_name The name of the CAN interface.
+   * @param node_id The ID of the node.
+   * @param node_name The name of the node.
+   *
+   * @exception std::runtime_error Thrown if can interface cannot be found.
+   * */
+  virtual void SetUavCanSettings(
```

Comment:
Could you add brief or change function name? I don't know what Set `Uav` mean.

Reply:
I would change it to `ConfigureUavCan`. `ConfigureCan`  wouble be more like changing a baudrate of can.